pagopa/io-app
# feat: [IOCOM-868] Update SEND message header and content to the new design system

## Short description
This PR updates the header and content of a SEND message.

_NOTE: Other PRs will follow to update the rest of the message._

<details open><summary>Details</summary>
<p>

| message details |
| - |
| <img src="https://github.com/pagopa/io-app/assets/29163287/728a25f9-a597-4543-b3d2-02ef46695c05" width="300" /> |

</p>
</details> 

## List of changes proposed in this pull request
- added `MessageDetailHeader`
- updated `MessageDetailsContent`
- added tests

## How to test
Using the `io-dev-api-server`, generate a SEND message. Enable the design system and navigate to the message detail. Check that the content is displayed correctly.
